scripts/ansible/pool_install.yml

```diff
@@ -4,9 +4,10 @@
   vars:
     pool_prefix: live_node
     pool_size: "{{ groups['pool']|count }}"
-    node_ver: 1.3.388
-    plenum_ver: 1.2.331
-    crypto_ver: 0.4.0
+    node_ver: 1.3.448
+    plenum_ver: 1.2.390
+    crypto_ver: 0.4.1
```

Comment:
Do you mean `python_indy_crypto_ver`? Why it's different from `libindy_crypto_ver`?